contracts/AllowanceTarget.sol

```diff
@@ -1,6 +1,5 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.7.6;
-
```

Comment:
This blank line should be kept, right?

Reply:
Yeah, I was trying to make it consistent across all contrasts. Thanks for the reminding. Fixed in 119500a